.storybook/useBrand.js

```diff
@@ -2,62 +2,10 @@ import { useEffect } from 'react';
 
 export const useBrand = (StoryFn, context) => {
     const {brand} = context.globals;
-    const {displayName} = context.component;
-    const components = [
```

Comment:
I'm out of the loop here, what's the goal of this removal?

Reply:
We did the rebrand everywhere, so we don't need to specify specific components anymore.